tests/unit/expiry/TestDataProvider.unitTest.ts

```diff
@@ -22,25 +23,31 @@ describe("TestDataProvider", () => {
           testDataProvider.testResultsDAO = new MockTestResultsDAO();
           await testDataProvider.getTestHistory("123");
         } catch (error) {
-          expect.assertions(1);
           expect(error).toEqual(
-            new HTTPError(500, MESSAGES.INTERNAL_SERVER_ERROR)
+            new HTTPError(500, enums.MESSAGES.INTERNAL_SERVER_ERROR)
           );
         }
       });
     });
-    describe("when getBySystemNumber returns invalid testEndTimestamp", () => {
+    describe("when getBySystemNumber returns invalid testStartTimestamp", () => {
       it("should throw error", async () => {
```

Comment:
could we add a bit more information when it should throw the error in the description please ?
Is that when `testEndTimestamp ` is incorrectly formatted?